qoe-metrics-dashboard/src/app/components/nav-bar/NavBar.tsx

```diff
@@ -1,7 +1,72 @@
-import React from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import { AppBar, Box, Container, Toolbar, useTheme } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+import logo from '../../../assets/Logo_5G_MAG.png';
+import './NavBar.scss';
+
+const pages = [
+  { label: 'Metrics Reports', route: '/metrics' },
+  { label: 'Consumption Reports', route: '/consumption' },
+];
+
+const CustomNavLink = styled(Link, {
+  name: 'MuiCustomNavLink',
+  slot: 'root',
+})<{ isactive: string }>(({ theme, isactive }) => ({
```

Comment:
please cemal case for isActive & type should be boolean
```suggestion
})<{ isActive: boolean }>(({ theme, isActive }) => ({
```

Reply:
Das funktioniert nicht, hatte ich zuerst auch. Man muss verhindern, dass das Prop an das DOM Element durchgereicht wird. Jetzt stimmt es